BrianJ-27/Cart-Ministry-Scheduler-v2
# add json-server package and a testable mock implementation

This adds json server which we will use to mock out our API calls during development.
To test it, run npm run start-api
You can then go to http://localhost:3001/messages to see it respond.
More details about json-server can be found at: https://www.npmjs.com/package/json-server
